### PR TITLE
Proposed tweaks to Leave Team PR #261

### PR DESCRIPTION
**Pull request by it33**

### CODE REVIEW

File: source/help/getting-started/managing-members.md [hunk 1]
```diff
@@ -48,30 +48,24 @@ The System Admin is typically a member of the IT staff and has all the privilege
 
 #### Inactive 
 
-A System Administrator has the ability to make a user account Inactive by going to **System Console > Teams**, and selecting a team to manage. From there they can go to the **User** page for the team, and manage a user's role. 
-
-When "Inactive" is selected, the user will no longer be able to log into the system. 
+A System Administrator can turn a user account Inactive by going to **System Console > Teams**, selecting a team to manage, the clicking on **User** and selecing "Make Inactive". Inactive users are removed from all teams and private groups and logged out from all devices and can no longer log in to the system. Users can be re-activated using the "Make Active" menu option.
```
Comment (lfbrock) [inline source/help/getting-started/managing-members.md]: "the" should be "then", "selecing" should be "selecting"

This is also incorrect? Inactive users aren't really removed from teams. If you turn them active again, they still belong to the team.

File: source/help/getting-started/managing-members.md [hunk 2]
```diff
@@ -48,30 +48,24 @@ The System Admin is typically a member of the IT staff and has all the privilege
...
+Team Administrators can remove a user from a team by going to **Main Menu > Manage Members**, and selecting **Remove From Team** in the dropdown menu beside a user entry. 
 
 When a user is removed from a team, if they currently have the team open they will be sent back to the team selection page. The team will no longer show up in their teams list. 
 
-Removing a user from the team does not make the account Inactive. The user will still be able to sign in to the site, and join other teams. They will also be able to rejoin the team they were removed from if they receive another invite, or if the team is an ["Open"](http://docs.mattermost.com/help/settings/team-settings.html#allow-anyone-to-join-this-team) team. If the user does rejoin the team, they will no longer belong to the channels they were previously a part of. 
+Removing a user from the team does not make the account Inactive. The user will still be able to sign in to the site, and join other teams. They will also be able to rejoin the team depending on the team's restrictions. If the user does rejoin the team, they will no longer belong to the channels they were previously a part of.
```
Comment (lfbrock) [inline source/help/getting-started/managing-members.md]: Is this really clear enough? It doesn't say what the restrictions are? 

Shouldn't we make it explicitly clear that if the team allows anyone to join, they can immediately rejoin if it's an open team.

**Comment by esethna** [comment]
@it33 see LB's comments above^

**Comment by lfbrock** [comment]
Closed and update the original PR based on the feedback instead.

**Pull request closed without merging**